### ISSUE TITLE
mWDNPlus wavelet selection error 

### ISSUE THREAD

**Issue by MOREDataset** [timeseriesAI/tsai#579]
Hi @oguiza, I get an error when I try to use the mWDN with a specified wavelet function for multi-step forecasting.
I tried a simple code as well and it still gives me the same error:
X, y, splits = get_UCR_data('LSST', split_data=False)
learn = TSForecaster(X, y, splits=splits, tfms=tfms, bs=512, arch=mWDNPlus, arch_config={'levels':15, 'wavelet':'db1', 'base_arch': LSTMPlus,**{'n_layers':5, 'bias':True, 'rnn_dropout':0.48641177153396, 'bidirectional':True,'fc_dropout':0.499669905155792, 'bn':False}})

Error found:
![image](https://user-images.githubusercontent.com/14005584/190281713-5bb5f62c-4266-41d0-9986-731fb4a55d99.png)

I tried installing the PYWT package first but for no avail.

**Comment by oguiza** [comment]
Hi @MOREDataset ,
To use ,WDNPlus you'll need to install PyWavelets>=1.1.1.
Have you tried installing tsai using: ```pip install tsai[extras]```?

**Comment by MOREDataset** [comment]
Hi @oguiza,
I tried both suggestions but none of them works. I still get the same error! A workaround is to manually change the `h_filter `and `l_filter `in the source code to reflect those of another wavelet.

**Comment by MOREDataset** [comment]
Hi @oguiza,

I have copied a snippet of the code from mWDN(Module) class in tsai/models/mWDN.py that defined the case of installing the pywt library in case it is not installed, and added it to the mWDNBlocks class:

`class mWDNBlocks(Module):
    def __init__(self, c_in, c_out, seq_len, levels=3, wavelet=None):
        self.levels=levels
        self.blocks = nn.ModuleList()
        if wavelet is not None:
            try:
                import pywt
            except ImportError:
                print("You need to either install pywt to run mWDN or leave wavelet=None")
        for i in range(levels): self.blocks.append(WaveBlock(c_in, c_out, seq_len // 2 ** i, wavelet=wavelet))
    def forward(self, x):
        for i in range(self.levels):
            x, out_ =  self.blocks[i](x)
            if i == 0: out = out_ if i == 0 else torch.cat((out, out_), dim=-1)
        return out`

Now this issue is resolved for me!

**Comment by oguiza** [comment]
Hi @MOREDataset,
Thanks for sharing this!
I've just updated the code following your approach (which makes more sense than what was implemented). 
So it should work well now without making any changes.
You'll need to install tsai from GitHub until the next pip/ conda release (0.3.6) becomes available.
